client/mainmenu/CHighScoreScreen.cpp

```diff
@@ -204,6 +206,12 @@ CHighScoreInputScreen::CHighScoreInputScreen(bool won, HighScoreCalculation calc
 		videoPlayer = std::make_shared<VideoWidgetOnce>(Point(0, 0), VideoPath::builtin("LOSEGAME.SMK"), true, [this](){close();});
 		CCS->musich->playMusic(AudioPath::builtin("music/UltimateLose"), false, true);
 	}
+
+	if (settings["general"]["enableUiEnhancements"].Bool())
+	{
+		statisticButton = std::make_shared<CButton>(Point(726, 10), AnimationPath::builtin("TPTAV02.DEF"), CButton::tooltip(CGI->generaltexth->translate("vcmi.statisticWindow.statistics")), [this](){ GH.windows().createAndPushWindow<CStatisticScreen>(stat); }, EShortcut::HIGH_SCORES_STATISTICS);
```

Comment:
Might be better to show statistics as a separate window, e.g. after win/loss video, but before high scores (if any).

This button is rather hard to find

Reply:
Yes. But I still prefer the extra button. As it is more like in original and not every time you want to see the extra screen with statistics.

Reply:
Original does not have any buttons on this screen.

And by the same logic - not every time you want to see win/loss movie, but it is unconditional.

Reply:
Hmm...

Still not very happy with it. We could add it:
- Before videos (not good in my opinion)
- Before highscore (but highscore input should still overlayed over videos -> makes no sense if you input your name -> then statistics -> then highscore)
- After highscore (possible; but should be accessible immediately after game, I think)

Still prefer the optional button. And I think the button is clearly visible, even when it's above the video.